### ISSUE TITLE
Published Nupkg Missing Dependencies

### ISSUE THREAD

**Issue by maldworth** [acarteas/FileCache#10]
Hey Sorry I didn't realize this earlier. Did you generate the Nuget package yourself to put into Nuget.org? Because it's missing the target framework dependencies. The Cake.build handles generation of the nupkg with all the correct versions and dependencies.

You can access the artifacts here: https://ci.appveyor.com/project/acarteas/filecache/build/2.1.0.build.7/artifacts

Because when I look at the nuspec for the one currently published:

```xml
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
  <metadata>
    <id>FileCache</id>
    <version>2.1.0</version>
    <authors>Adam Carter</authors>
    <owners>Adam Carter</owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <licenseUrl>https://github.com/acarteas/FileCache/blob/master/LICENSE</licenseUrl>
    <projectUrl>https://github.com/acarteas/FileCache</projectUrl>
    <description>FileCache is a concrete implementation of the .Net Framework 4's System.Runtime.Caching.ObjectCache that uses the local filesystem as the target location.  FileCache now supports .NET Standard.</description>
    <summary></summary>
    <tags>cache objectcache System.Runtime.Caching.ObjectCache</tags>
  </metadata>
</package>
```

versus the one from the appveyor artifacts:

```xml
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
  <metadata>
    <id>FileCache</id>
    <version>2.1.1</version>
    <authors>Adam Carter</authors>
    <owners>Adam Carter</owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <licenseUrl>https://github.com/acarteas/FileCache/blob/master/LICENSE</licenseUrl>
    <projectUrl>https://github.com/acarteas/FileCache</projectUrl>
    <description>FileCache is a concrete implementation of the .NET System.Runtime.Caching.ObjectCache that uses the local filesystem as the target location.</description>
    <copyright>Copyright (c) 2012, 2013, 2017 Adam Carter (http://adam-carter.com)</copyright>
    <tags>cache objectcache System.Runtime.Caching.ObjectCache</tags>
    <dependencies>
      <group targetFramework=".NETFramework4.0" />
      <group targetFramework=".NETFramework4.5" />
      <group targetFramework=".NETStandard2.0">
        <dependency id="System.Runtime.Caching" version="4.5.0-preview2-26406-04" exclude="Build,Analyzers" />
      </group>
    </dependencies>
    <frameworkAssemblies>
      <frameworkAssembly assemblyName="System.Runtime.Caching" targetFramework=".NETFramework4.0, .NETFramework4.5" />
    </frameworkAssemblies>
  </metadata>
</package>
```

it's missing the dependencies group. You might need to unlist the current 2.1.0, and then use the artifacts from [2.1.1](https://ci.appveyor.com/project/acarteas/filecache/build/2.1.1+1.build.8/artifacts).

If you are also curious, you can run `.\build.ps1` on your local dev box and it will perform the exact same build that appveyor does, and it creates an \artifacts\ folder which has the nupkgs, which you can review as well.

**Comment by acarteas** [comment]
Oh, that's cool.  When I built it last time I didn't see those
auto-generated packages - maybe it takes a few minutes to build.  I'll be
more patient next time.

I'll create a 2.1.1 release tomorrow and push it.  I'm curious if the 2.1
packages work for you?  Before I published, I installed the nupkg on a test
project locally and the FC seemed to be working just fine so I figured
everything must've been good to go.

**Comment by maldworth** [comment]
Yep, the 2.1.0 package worked, however if its used by a project referencing netcoreapp2.0/netstandard2.0 it will not add the netstandard2.0 System.Runtime.Caching package, and will give a compiler warning. You can ignore the warning and it works, but it's probably best to have the dependencies specified within the nuspec :).

Thanks!

**Comment by acarteas** [comment]
It turns out that Nuget won't let you label a release as "stable" if it relies on an alpha library, which seems a bit silly.  So, I ended up having to omit that from the nupkg.  I might add a blurb in the readme and push a 2.1.2 when they come out with a stable version of .NET caching for .NET standard.

**Comment by maldworth** [comment]
Ohh I didn't know. Is there any chance you could publish a pre-release version for me? I actually need the netstandard version.  I believe you'd need to just use this package: https://ci.appveyor.com/project/acarteas/filecache/build/2.2.0-alpha.2.build.10/artifacts

and then it would keep your 2.1.0 version, but have a 2.2.0-alpha.2 prerelease available for people who want to test netstandard.

**Comment by acarteas** [comment]
Okay, I just published.

**Comment by maldworth** [comment]
Fantastic this works.  You should however Unlist 2.1.1, it actually is broken. Just keep:
- 2.1.0
- 2.2.0-alpha0002

and when .net core 2.1 is release (they have said Q2 this year I think), then i assume the System.Runtime.Caching will be fully released, and then you could make a full 2.2.0 release with it.
